src/reporters/screenshot.reporter.ts

```diff
@@ -8,12 +9,13 @@ export class ScreenshotReporter {
         this.basePath = basePath;
     }
 
-    public static beforeSuite(suite: any): void {
+    public static beforeSuite(suite: any = null): void {
```

Comment:
Are this default values required now that we are using a reporter to call this?

Reply:
not required, reverted